modules/nix/default.nix

```diff
@@ -678,6 +678,7 @@ in
       "c6080216f2a170611e339c3f46e4e1d61aaf0d8b417ad93ade8d647da1382c11"  # DeterminateSystems Nix installer 0.14.0
       "97f4135d262ca22d65c9554aad795c10a4491fa61b67d9c2430f4d82bbfec9a2"  # DeterminateSystems Nix installer 0.15.1
       "5d23e6d7015756c6f300f8cd558ec4d9234ca61deefd4f2478e91a49760b0747"  # DeterminateSystems Nix installer 0.16.0+
```

Comment:
Could you update this with the version that this configuration file stopped being used? I think it would be the version when they added the `upgrade-nix-store-path-url` default setting

Reply:
Yes, I'll see to it tomorrow

Reply:
Seems it was in v0.16.0 according to the commit it was added in:

* https://github.com/DeterminateSystems/nix-installer/commit/0b0172547c4666f6b1eacb6561a59d6b612505a3
* https://github.com/DeterminateSystems/nix-installer/compare/v0.15.1...v0.16.0

Reply:
> Yes, I'll see to it tomorrow

Hi everyone, sorry for going ghost on this issue. I went on vacation and was AFK, forgot this was active. Thanks for closing this with a fix